File: challenges/1-produto-simples/typescript/Juninhoww2/produtoSimples.ts
const valor1 = Number(process.argv.slice(2, 3));
const valor2 = Number(process.argv.slice(3, 2));
const produto = produtoSimples(valor1, valor2);

console.log(`produto: ${produto}`);

function produtoSimples(valor1: number, valor2: number): number {
    return valor1 * valor2;
}

File: challenges/1-produto-simples/java/Neg0023/ProdutoSimples.java

public class ProdutoSimples {
    public static void main(String[] args) {
        System.out.printf("produto: %d", produto(Integer.parseInt(args[0]), Integer.parseInt(args[1])));
    }

    private static int produto(int a, int b) {
        return a * b;
    }
}
